psi4/src/read_options.cc

```diff
@@ -2633,8 +2631,8 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         options.add_double("IRC_STEP_SIZE", 0.2);
         /*- IRC mapping direction -*/
         options.add_str("IRC_DIRECTION", "FORWARD", "FORWARD BACKWARD");
-        /*- Decide when to stop IRC calculations -*/
-        options.add_str("IRC_STOP", "STOP", "ASK STOP GO");
+        /*- Number of IRC points to collect before stopping -*/
+        options.add_int("IRC_POINTS", 20);
```

Comment:
Yes, there's an energy-minimum check at each converged point; it'll stop if it finds one.

Reply:
ok, gtk. I originally asked to see if it might fit as a "MAXITER" keyword. But I now don't think that's right anyhow. With a SCF or OPT maxiter, if it converges in 2, you're pleased. Here, if it converges in 2, you have only a meager IRC.

Reply:
To clarify, at least in my mind, its most helpful when dealing with paths like a dissociation channel where no well defined minimum may exist to keep optking from doing unnecessary, unending work. It can also function as a distance metric i,.e. do 30 points or 3 Angstroms along the reaction path. If the irc doesn't end before 20 points, the user can just restart anyhow.